internal/ui/playlists.go

```diff
@@ -70,21 +62,21 @@ func (playlistModel PlaylistModel) View() string {
 	return docStyle.Render(playlistModel.list.View())
 }
 
-func NewPlaylistModel() PlaylistModel {
-	playlists := api.GetPlaylists()
+func NewPlaylistModel() *PlaylistModel {
+	var playlists []list.Item
 
-	for _, playlist := range playlists {
-		playlistItems = append(playlistItems, playlistItem{
-			id:   string(playlist.ID),
-			name: playlist.Name,
+	for _, p := range api.GetPlaylists() {
+		playlists = append(playlists, playlist{
```

Comment:
Why not declare new `playlist` as a pointer here? i think it would make since to make sure that we're always dealing with the same instance